### PR TITLE
DOC: add docs for folder tag list

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@sbillinge @chiahaoliu can I get some review?

File: docs/templates/usb_Running.rst [hunk 1]
```diff
@@ -718,6 +718,10 @@ like ``folder_tag``. When ``folder_tag`` is added
 (eg ``xrun(0, 8, folder_tag='after_saxs')`` then the ``after_saxs`` will be
 added to the file path where the data is saved
 (eg ``<.../tiff_base/sample_name/after_saxs/mask/file_name>``).
+You can also define multiple folder tags by supplying ``folder_tag_list``
+(eg ``xrun(0, 8, folder_tag_list=['status', 'sample_name', bt_piLast'], status='after_saxs')``
+adding the values of the metadata for those keys into the file path
+(eg ``<.../tiff_base/after/saxs/sample_name/bt_piLast/mask/file_name>``).
```
Comment (chiahaoliu) [inline docs/templates/usb_Running.rst]: is ``folder_tag_list`` a list of keys which defines the mapping of actual directory structure? Like ``sample_name`` is ``Ni`` and ``bt_pilast`` is ``Billinge`` for a certain scan and those fields are different for other scans. 

If this is the case, can we add ``<>`` around the keys which will be parsed later? and maybe we need some short explanation for this logic.

Also, in this example, shouldn't the folder name be:
``<.../tiff_base/after_saxs/...>``

Comment (CJ-Wright) [reply docs/templates/usb_Running.rst]: Yes.

What do you mean "`<>` around the keys..."?

Yes, that is a typo.

Comment (CJ-Wright) [reply docs/templates/usb_Running.rst]: Ah ok, I've updated the example, does it do a better job explaining now?

**Pull request merged**